spread.yaml

```diff
@@ -22,8 +22,6 @@ backends:
                 image: ubuntu-os-cloud/ubuntu-1804-lts
             - ubuntu-19.04-64:
                 image: ubuntu-os-cloud/ubuntu-1904
-            - ubuntu-devel-64:
-                image: ubuntu-os-cloud/ubuntu-1810
```

Comment:
This is actually a typo. This should say `1910` instead.

Reply:
I tried that and it didn't work: https://travis-ci.org/MirServer/mir/jobs/564109665

```
2019-07-26 17:10:09 Allocating google:ubuntu-devel-64...
2019-07-26 17:10:09 Cannot allocate google:ubuntu-devel-64: cannot find any Google image matching "ubuntu-os-cloud/ubuntu-1910" on project "ubuntu-os-cloud"
2019-07-26 17:10:09 Successful tasks: 0
2019-07-26 17:10:09 Aborted tasks: 1
error: unsuccessful run
```

Reply:
Make this `1904` then, it will get upgraded to devel in CI.